src/main/kotlin/com/hltech/judged/contracts/publisher/judged/JudgeD.kt

```diff
@@ -0,0 +1,27 @@
+package com.hltech.judged.contracts.publisher.judged
+
+import com.hltech.judged.contracts.publisher.capabilities.Capabilities
+import feign.Feign
+import feign.jackson.JacksonDecoder
+import feign.jackson.JacksonEncoder
+import org.gradle.api.Project
+
+class JudgeD {
+
+    fun publishContracts(project: Project, capabilities: List<Capabilities>) {
+        val capabilitiesMap = capabilities
+            .map { it.communicationInterface to ContractForm(it.value, it.mimeType) }
+            .toMap()
+
+        val contractsForm = ServiceContractsForm(capabilitiesMap, emptyMap())
+
+        val judgeDLocation = project.properties["judgeDLocation"]?.toString()
+            ?: throw IllegalArgumentException("judgeDLocation parameter is required to publish contracts")
```

Comment:
`"Parameter 'judgeDLocation' is required...` will better highlight the name of parameter